charts/external-secrets/templates/govuk-chat/bigquery.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: govuk-chat-bigquery
+  labels:
+    {{- include "external-secrets.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/description: >
+      Credentials for connecting to GCP BigQuery project.
+spec:
+  refreshInterval: {{ .Values.externalSecrets.refreshInterval }}
+  secretStoreRef:
+    name: aws-secretsmanager
+    kind: ClusterSecretStore
+  target:
+    deletionPolicy: {{ .Values.externalSecrets.deletionPolicy }}
+  dataFrom:
+    - extract:
+        key: govuk/govuk-chat/bigquery
```

Comment:
Nit: you missed the trailing new line here - you can configure your text editor to include one automatically. If you're using VS Code: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
I don't quite understand why this was missing. I had a trailing new line in the commit locally.

I've just added a second one locally and pushed that, and now it's showing with a single new line in github. It only seems to be an issue in this file though.

Reply:
Oh that's annoying!